rust/relay/src/lib.rs

```diff
@@ -23,6 +23,11 @@ use std::{
     net::{IpAddr, Ipv4Addr, Ipv6Addr, SocketAddr},
 };
 
+pub const VERSION: &str = match option_env!("GITHUB_SHA") {
```

Comment:
Does `rust/Dockerfile` need updating to pass this as a build arg? CI configuration may need updating as well.

The `elixir/Dockerfile` specifies a `GIT_SHA` arg that we populate from `GITHUB_SHA` in the CI workflow for example.

Reply:
This tag has been working before already. We build the release binaries outside the docker image and copy them in.